sql/7_0_0-to-7_0_1_upgrade.sql

```diff
@@ -201,3 +201,7 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 #IfNotColumnType prescriptions route VARCHAR(100)
 ALTER TABLE `prescriptions` CHANGE `route` `route` VARCHAR(100) NULL DEFAULT NULL Comment 'Max size 100 characters is same max as immunizations';
 #EndIf
+
+#IfMissingColumn layout_group_properties grp_unchecked
+ALTER TABLE `layout_group_properties` ADD `grp_unchecked` tinyint(1) NOT NULL DEFAULT 0;
```

Comment:
also need database change in database.sql
(then I can make an Up For Grabs demo from this PR so folks can quickly test it out :) )